### PR TITLE
add august edition

### PR DESCRIPTION
**Pull request by flashadvocate**

### CODE REVIEW

File: TOC.md [hunk 1]
```diff
@@ -5,6 +5,10 @@
   * February: [raw](2018/02.md) | [slides](https://speakerdeck.com/flashadvocate/latest-in-php-february-2018-edition)
   * March: [raw](2018/03.md) | [slides](https://speakerdeck.com/flashadvocate/latest-in-php-march-2018-edition)
   * April: [raw](2018/04.md) | [slides](https://speakerdeck.com/cmgmyr/the-latest-in-php-april-2018-edition)
+  * May: MIA!
+  * June: MIA!
+  * July: MIA!
+  * August: [raw](2018/03.md) | [slides](https://speakerdeck.com/flashadvocate/latest-in-php-august-edition)
```
Comment (cmgmyr) [inline TOC.md]: `08.md` 😉

**Review by cmgmyr — CHANGES_REQUESTED** [review]
quick change to the TOC

**Comment by flashadvocate** [comment]
Oops, sorry about that. Fixed :)

**Review by cmgmyr — APPROVED** [review]
Thanks!

**Pull request merged**